backend/controllers/prisonerProfile/personalViewModels/careNeedsViewModel.ts

```diff
@@ -36,7 +36,7 @@ export default ({ personalCareNeeds, reasonableAdjustments, treatmentTypes, heal
             details: [
               {
                 label: 'Establishment',
-                value: agencyDetails && (agencyDetails.formattedDescription || agencyDetails.description),
+                value: agencyDetails && agencyDetails.description,
```

Comment:
on the different endpoint `description` is always populated with the formatted description